collectors/bzimport/tests/test_convertors.py

```diff
@@ -233,6 +234,7 @@ def test_basic(self):
         assert flaw.package_versions.first() == package_versions
         assert flaw.major_incident_state == Flaw.FlawMajorIncident.REQUESTED
         assert flaw.requires_summary == Flaw.FlawRequiresSummary.NOVALUE
+        assert flaw.nist_cvss_validation == Flaw.FlawNistCvssValidation.REJECTED
```

Comment:
I think there is an issue in `FlawFactory`. The `nist_cvss_validation` field is not conditional on the `cvss` field. This way, you can create a flaw without `cvss` but with `nist_cvss_validation=REJECTED`, which is not correct. 

Btw. `requires_summary` probably has a similar problem.

Reply:
This should be solved in a different PR. You can ignore the comment above.